htdocs/main.inc.php

```diff
@@ -443,7 +443,7 @@ function analyseVarsForSqlAndScriptsInjection(&$var, $type, $stopcode = 1)
 	global $debugbar;
 	include_once DOL_DOCUMENT_ROOT.'/debugbar/class/DebugBar.php';
 	$debugbar = new DolibarrDebugBar();
-	$renderer = $debugbar->getRenderer();
+	$renderer = $debugbar->getJavascriptRenderer();
```

Comment:
Do we really need to replace getRenderer with getJavascriptRenderer() ?
What about the call of disableVendor() that is inside getRender() that is lost ?

Reply:
This change is because of
https://github.com/Dolibarr/dolibarr/blob/64ecea7c83db5e4a433b07fc94295e9f3e240028/htdocs/core/lib/functions.lib.php#L10800-L10804

That code supposes that the DebugBar has a getRenderer) method which it does not have.

The DolibarrDebugBar was updated such that a call to the getJavascriptRenderer without arguments does exactly the same as the original getRenderer() method which is maintained as a deprecated method:

https://github.com/Dolibarr/dolibarr/blob/4547c8306077e9ca7e3e811d834200240f7b48d5/htdocs/debugbar/class/DebugBar.php#L77-L89